src/pages/FaqPage.tsx

```
@@ -0,0 +1,57 @@
+                  <AccordionContent>
+                    <div 
+                      className="prose dark:prose-invert max-w-none
+                               prose-headings:font-bold prose-headings:text-foreground
+                               prose-h2:text-2xl prose-h3:text-xl
+                               prose-p:text-base prose-p:leading-relaxed prose-p:text-foreground/90
+                               prose-a:text-primary prose-a:no-underline hover:prose-a:underline
+                               prose-strong:text-foreground prose-strong:font-semibold
+                               prose-li:marker:text-primary/70 prose-li:my-1 prose-li:text-foreground/90
+                               prose-img:rounded-md prose-img:shadow-sm
+                               prose-hr:border-border/40"
+                      dangerouslySetInnerHTML={{ __html: item.answer }} 
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)